test_scripts/Defects/4_5/1873_Parameters_empty_in_policy_table.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------
+-- In case
+-- mobile app sends request to SDL
+-- and this request is allowed by Policies for this mobile app
+-- and "parameters" field is empty at PolicyTable for RPC
+-- SDL must:
+-- respond with "DISALLOWED, success:false", "info: Requested parameters are disallowed by Policies"
+-- SDL must NOT:
+-- transfer this request to HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonDefects = require('test_scripts/Defects/4_5/commonDefects')
+local json = require("json")
+
+--[[ Local Functions ]]
+local function ptuUpdateFuncDissalowedRPC(tbl)
+  local VDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = json.EMPTY_ARRAY
+      },
+      SubscribeVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = json.EMPTY_ARRAY
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
```

Comment:
@GetmanetsIrina I would propose to use dot notation to access to table fields instead of square brackets notation in cases when field name is string literal.
Example:
`tbl.policy_table.functional_groupings.NewTestCaseGroup = VDgroup `
instead of
`tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup `
but 
`tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups = {"Base-4", "NewTestCaseGroup"}`